packages/react-vapor/src/components/slider/SliderUtils.ts

```diff
@@ -0,0 +1,62 @@
+export const valuesPositionOnRange = {
+    lower: 'LOWER_THAN_CROSSING_POINT',
+    higher: 'HIGHER_THAN_CROSSING_POINT',
+    both: 'NO_VALUE_AT_CROSSING_POINT',
+};
+
+export const getValuesPositionOnRange = (value: number[], crossingPoint: number): string => {
+    if (value.every((singleValue) => singleValue < crossingPoint)) {
+        return valuesPositionOnRange.lower;
+    } else if (value.every((singleValue) => singleValue > crossingPoint)) {
+        return valuesPositionOnRange.higher;
+    } else if (value.every((singleValue) => singleValue !== crossingPoint)) {
+        return valuesPositionOnRange.both;
+    }
+    return null;
```

Comment:
what if every single value is at the crossing point?

Reply:
both handles should render and overlaps thus giving the feel of a single handle.